Libraries/LibWeb/CSS/CSSStyleProperties.cpp

```diff
@@ -185,7 +185,7 @@ Optional<StyleProperty> CSSStyleProperties::property(PropertyID property_id) con
         }
 
         if (!layout_node) {
-            auto style = element.document().style_computer().compute_style(element, pseudo_element);
+            auto style = element.document().style_computer().compute_style(element, pseudo_element, {}, false);
```

Comment:
After a short discussion in the #css channel; adding an argument to disable an internal functionality/optimization like this is not really a well-thought out API.

It would be better to support `compute_style()` for single elements without tree traversal. For example, you could do a short ancestor traversal on the spot so you don't need to disable the filter at all.